app/controllers/events_controller.rb

```diff
@@ -0,0 +1,51 @@
+# Events controller class
+class EventsController < ApplicationController
+
+  before_action :set_event, only: [:show, :edit, :update, :detroy ]
+
+  def index
+    @events = Event.all
+  end
+
+  def new
```

Comment:
All creating logic will be perfromed from the admin panel. No need of having these actions in this controller.
Just an index and show action will be enough.